deployments/src/abis.ts

```diff
@@ -0,0 +1,103 @@
+import Assessment from '../generated/abis/Assessment.json';
```

Comment:
Is it possible to generate this file on the fly?

Reply:
Not as easy as we do with other files. Since this is a `.ts` file, it tells typescript/rollup which modules are imported. This info will be used to determine what to treeshake away and what to bundle. So we would need to generate this before running typescript, which is a little too much complexity for what it's worth imo.

What we can do to reduce the maintenance burden is to make a script that generates and overwrites this file for us. This way we can have it in source control, but we can re-generate to check/update anytime we like 🤔

Reply:
> What we can do to reduce the maintenance burden is to make a script that generates and overwrites this file for us

That's exactly what I meant! `pre-publish.js` seems a good place for this.

Reply:
Good idea, added!